build/dependencies.props

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
   <!-- Partner Versions-->
   <PropertyGroup>
-    <CodeAnalysisVersion>2.3.0-beta1</CodeAnalysisVersion>
+    <CodeAnalysisVersion>2.3.0-beta3-61731-06</CodeAnalysisVersion>
```

Comment:
Confirmed with Roslyn and CLI folks that the right version here is `2.3.0-beta3-61814-09`